scripts/populate_annex_scientificname.py

```diff
@@ -11,25 +13,32 @@ def _get_annex(path):
     """
     with open(path) as csvfile:
         annex_data = reader(csvfile)
-        scientificnames_annex = dict()
+        annex_scientificnames = dict()
         fields = next(annex_data)
         print("Columns in " + path + ": " + ", ".join(fields))
         for (i, row) in enumerate(annex_data):
             id = i + 1
             scientific_name_original = row[1]
             scientific_name_corrected = row[2]
+            authorship = row[3]
             annex_id = row[0]
-            remarks = row[4]
-            scientificnames_annex[id] = {'id': id,
-                                         'scientificNameOriginal': scientific_name_original,
+            remarks = row[5]
+            is_scientific_name = True
+            if scientific_name_corrected == '':
+                is_scientific_name = False
+            annex_scientificnames[id] = {'id': id,
+                                         'scientificNameId': None,
+                                         'scientificNameInAnnex': scientific_name_original,
                                          'scientificName': scientific_name_corrected,
+                                         'authorship': authorship,
+                                         'isScientificName': is_scientific_name,
                                          'annexCode': annex_id,
                                          'remarks': remarks}
-    return scientificnames_annex
+    return annex_scientificnames
 
 
-def populate_scientificname_annex(conn, config_parser, annex_file):
-    """ Populate the table scientificnameannex
+def populate_annex_scientificname(conn, config_parser, annex_file):
```

Comment:
(_this is a general comment, not to solved in this PR, but maybe useful for future developments or for a cleanup step_):

I notice some functions/loops like this one are quite long. I think it would be more readable to make them shorter, when possible. A few possible approaches in this case:

- Divide in smaller functions with a small and limited scope
- The code is sometimes longer and more difficult to read because the actual data logic is interleaved with logic to print/log what happens and code to measure execution time, I wonder if we can do something to minimize this. 

For the specific case of timing function calls, now that you're more and more comfortable with Python, you might be interested to have a look at [decorators](https://realpython.com/primer-on-python-decorators/). I think they can work well for this, see for example discussions and examples at https://stackoverflow.com/questions/51503672/decorator-for-timeit-timeit-method, https://stackoverflow.com/questions/34981602/logging-execution-time-with-decorators, https://medium.com/pythonhive/python-decorator-to-measure-the-execution-time-of-methods-fa04cb6bb36d, ...